src/utils/date.js

```diff
@@ -107,6 +107,58 @@ export function formatDate(val, dateStyle = 'short', timeStyle = 'medium', local
 	);
 }
 
+export function formatDateCustomTooltip(val, dateFormat) {
+	//if relative is displayed use full as the tooltip
+	if(dateFormat == 'relative'){
```

Comment:
Please install 'prettier' plugin for your IDE and format new code. There is a config file in root, so it will apply automatically